### PR TITLE
Update cache-busting cookie patterns section of VCL snippet

### LINKED ISSUE #873: Varnish Caching - Drupal and WordPress Advanced Topics Doc Update
Re: [Varnish Caching - Drupal and WordPress Advanced Topics](https://pantheon.io/docs/articles/sites/varnish/caching-advancedtopics)
## Issue Description:
The last section with the cookie patterns is not up to date. The latest version from the Varnish VCL is below.

```
       'NO_CACHE',
       'S+ESS[a-z0-9]+',
       'fbs[a-z0-9_]+',
       'SimpleSAML[A-Za-z]+',
       'SimpleSAML[A-Za-z]+',
       'PHPSESSID',
       'wordpress[A-Za-z0-9_]*',
       'wp-[A-Za-z0-9_]+',
       'comment_author_[a-z0-9_]+',
       'duo_wordpress_auth_cookie',
       'duo_secure_wordpress_auth_cookie',
       'bp_completed_create_steps', # BuddyPress cookie used when creating groups
       'bp_new_group_id', # BuddyPress cookie used when creating groups
       'wp-resetpass-[A-Za-z0-9_]+',
       '(wp_)?woocommerce[A-Za-z0-9_-]+'
```

### PR DESCRIPTION
**Pull request by bmackinney**
closes #873

### CODE REVIEW

File: source/docs/articles/sites/varnish/caching-advancedtopics.md [hunk 1]
```diff
@@ -142,7 +142,7 @@ If you're using the Environment Access: Locked security setting on a site envir
 
 ## Pantheon's .vcl File
```
Comment (ari-gold) [inline source/docs/articles/sites/varnish/caching-advancedtopics.md]: Also change heading? Maybe  to "Pantheon's Varnish cookie handling" like in https://github.com/pantheon-systems/documentation/pull/301

Comment (bmackinney) [inline source/docs/articles/sites/varnish/caching-advancedtopics.md]: @ari-gold I thought about that. Any links to the existing heading would need to be changed, unless we can redirect to a different heading in the same document. The last time I tried that it failed in Drupal.

Comment (ari-gold) [inline source/docs/articles/sites/varnish/caching-advancedtopics.md]: @bmackinney Ah, that's a bummer. How about creating a separate issue for that? I think there's more value in having an accurate section heading than redirecting to a specific heading. As long as people land on the right page, I think they should be able to find what they need. What do you think?

Comment (bmackinney) [inline source/docs/articles/sites/varnish/caching-advancedtopics.md]: @ari-gold you're right. making the change.

**Comment by ari-gold** [comment]
LGTM :+1:

**Pull request merged**